addons/gdUnit4/runtest.sh

```diff
@@ -6,7 +6,7 @@ if [ -z "$GODOT_BIN" ]; then
     exit 1
 fi
 
-$GODOT_BIN --path . -s -d ./addons/gdUnit4/bin/GdUnitCmdTool.gd $*
+$GODOT_BIN --headless --path . -s -d ./addons/gdUnit4/bin/GdUnitCmdTool.gd $*
```

Comment:
The `--headless` mode is currently forbidden to use. Godot has still a bug with `headless` and mouse event processing.

https://github.com/godotengine/godot/issues/73557